### ISSUE TITLE
AVT 3 - Tabs: Tab content of tab 1 and 2 are not announced by VoiceOver

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2487]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Cannot navigate using the keyboard with Voiceover to tab 1 or tab 3 content. Tab 4 content is announced only after VoiceOver announces it has left the Tab navigation. Also, see issue #2171. 

## Expected Result
VO should announce each tab using the Right and Left Arrow keys. After a Tab is selected the Tab key should navigate to the tab content, as well as tab back to the original tab. See ARIA Authoring Practices 1.1 - Tabs for additional ARIA requirements.

**Comment by dakahn** [comment]
@snidersd I can't find an example of tabs that has this behavior you're describing. looking here
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

the tabs are activatable, but the content is not read automatically. i has to receive focus. Do I have this right?

**Comment by snidersd** [comment]
@dakhn I just retested with VO and I'm not seeing the issue anymore.
